bpf/lib/trace.h

```diff
@@ -189,13 +190,26 @@ send_trace_notify(struct __ctx_buff *ctx, enum trace_point obs_point,
 	__u64 ctx_len = ctx_full_len(ctx);
 	__u64 cap_len = min_t(__u64, monitor ? : TRACE_PAYLOAD_LEN,
 			      ctx_len);
+	struct ratelimit_key rkey = {
```

Comment:
datapath isn't my area of expertise, but should these be behind a `#ifdef` similar to other dp features?

Reply:
This was answered this in another review comment thread: https://github.com/cilium/cilium/pull/29711/files#r1433268232

resolving.